templates/terraform/extra_schema_entry/firebase_project.go.erb

```diff
@@ -0,0 +1,57 @@
+"resources": {
```

Comment:
I don't understand why you have defined this schema both in the api.yaml and this extra schema. You should only need to define it in the api.yaml

Reply:
The issue is that the "location_id" is really a computed property of the Firebase Project. The only way to change this property is to call the separate "projectLocation:finalize" endpoint.  Since I want the "location_id" property to be "Optional" in Terraform but never actually used on create, there is no way I can express that in the api.yaml.  

The only way was to mark the property as output in the api yaml (as is the resources Array) and then in the terraform.yaml, exclude the resources property all together and define custom code for the schema.